packages/teleport/src/services/database/databases.test.ts

```diff
@@ -33,6 +33,31 @@ test('correct formatting of database fetch response', async () => {
   ]);
 });
 
+describe('correct formatting of all type and protocol combos', () => {
+  test.each`
+    type                 | protocol                 | combined
+    ${'self-hosted'}     | ${'mysql'}               | ${'Self-hosted MySQL'}
+    ${'rds'}             | ${'mysql'}               | ${'RDS MySQL'}
+    ${'gcp'}             | ${'mysql'}               | ${'Cloud SQL MySQL'}
+    ${'redshift'}        | ${'mysql'}               | ${'Redshift MySQL'}
+    ${'self-hosted'}     | ${'postgres'}            | ${'Self-hosted PostgreSQL'}
+    ${'rds'}             | ${'postgres'}            | ${'RDS PostgreSQL'}
+    ${'gcp'}             | ${'postgres'}            | ${'Cloud SQL PostgreSQL'}
+    ${'redshift'}        | ${'postgres'}            | ${'Redshift PostgreSQL'}
+    ${'some other type'} | ${'some other protocol'} | ${'some other type some other protocol'}
+  `(
+    'should combine type and protocol correctly',
```

Comment:
if i remember correctly, when we run this test, the output will be:
```
correct formatting of all type...
   should combine
   should combine
   ....
```

There is no differentiating messages for each test, so when it fails, i won't know what test fails. global search for `test.each` and you can see how to access make a more unique message per test

Reply:
It does show which test fails, for example:

![image](https://user-images.githubusercontent.com/56373201/116476325-13036280-a849-11eb-8732-79cdfa75c648.png)

I can still add the unique message per test for convenience though